### ISSUE TITLE
Update Czech translations

### ISSUE THREAD

**Issue by 3p1463k** [squidfunk/mkdocs-material#5129]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "cs",
  "action.edit": "Upravit tuto stránku",
  "action.skip": "Přeskočit obsah",
  "action.view": "Zobrazit zdroj této stránky",
  "announce.dismiss": "Již nezobrazovat",
  "blog.archive": "Archiv",
  "blog.categories": "Kategorie",
  "blog.categories.in": "v",
  "blog.continue": "Pokračovat ve čtení",
  "blog.draft": "Návrh",
  "blog.index": "Zpět na index",
  "blog.meta": "Metadata",
  "blog.references": "Související odkazy",
  "clipboard.copy": "Kopírovat do schránky",
  "clipboard.copied": "Zkopírováno do schránky",
  "consent.accept": "Akceptovat",
  "consent.manage": "Spravovat nastavení",
  "consent.reject": "Odmítnout",
  "footer": "Zápatí",
  "footer.next": "Další",
  "footer.previous": "Předchozí",
  "header": "Záhlaví",
  "meta.comments": "Komentáře",
  "meta.source": "Zdroj",
  "nav": "Navigace",
  "readtime.one": "1 min čtení",
  "readtime.other": "# min čtení",
  "rss.created": "RSS kanál",
  "rss.updated": "RSS zdroj aktualizovaného obsahu",
  "search": "Vyhledávání",
  "search.placeholder": "Hledat",
  "search.share": "Sdílet",
  "search.reset": "Vyčistit",
  "search.result.initializer": "Inicializace vyhledávání",
  "search.result.placeholder": "Pište co se má vyhledat",
  "search.result.none": "Nenalezeny žádné dokumenty",
  "search.result.one": "Nalezený dokument: 1",
  "search.result.other": "Nalezené dokumenty: #",
  "search.result.more.one": "1 další na této stránce",
  "search.result.more.other": "# více na této stránce",
  "search.result.term.missing": "Chybějící",
  "select.language": "Zvolte jazyk",
  "select.version": "Vyberte verzi",
  "source": "Přejít do repozitáře",
  "source.file.contributors": "Přispěvatelé",
  "source.file.date.created": "Vytvořeno",
  "source.file.date.updated": "Poslední aktualizace",
  "tabs": "Karty",
  "toc": "Obsah",
  "top": "Zpět na začátek"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Thanks! Added in https://github.com/squidfunk/mkdocs-material/commit/48b89f3dc0796d5a2982c10635cd65fb5de4c3d2.

**Comment by 3p1463k** [comment]
Thank you.

**Comment by squidfunk** [comment]
Released as part of [9.1.1](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.1).
